apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -889,6 +912,20 @@ NetSimPacketEditor.prototype.setMaxPacketSize = function (maxPacketSize) {
 NetSimPacketEditor.prototype.setEncodings = function (newEncodings) {
   this.enabledEncodings_ = newEncodings;
   NetSimEncodingControl.hideRowsByEncoding(this.rootDiv_, newEncodings);
+  this.render();
+};
+
+/**
+ * Helper method that checks this.enabledEncodings_ to see if the given
+ * encoding is enabled
+ * @param {EncodingType} queryEncoding
+ * @returns {boolean} whether or not the given encoding is enabled
+ * @private
+ */
+NetSimPacketEditor.prototype.isEncodingEnabled_ = function (queryEncoding) {
+  return this.enabledEncodings_.some(function (enabledEncoding) {
```

Comment:
How about making enabledEncodings a hash rather than an array? Then you can just say "enabledEncoding[e]" instead of having to search the array repeatedly.  (Shorter and maybe slightly faster, though for such a small number of encodings the speed difference may not be obvious.)